dm/loader/lightning.go

```diff
@@ -161,25 +192,34 @@ func (l *LightningLoader) restore(ctx context.Context) error {
 			TLS:              cfg.TiDB.TLS,
 		}
 		cfg.Checkpoint.DSN = param.ToDSN()
+		cfg.TiDB.Vars = make(map[string]string)
+		if l.cfg.To.Session != nil {
+			for k, v := range l.cfg.To.Session {
+				cfg.TiDB.Vars[k] = v
+			}
+		}
+		cfg.TiDB.Vars["time_zone"] = "+00:00"
```

Comment:
timezone should be added to `l.cfg.To.Session`, so no need to manually adjust here